src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts

```diff
@@ -232,6 +356,82 @@ declare namespace Uno.UI {
         private getIsConnectedToRootElement(element);
     }
 }
+declare class WindowManagerAddViewParams {
+    HtmlId: number;
+    ChildView: number;
+    Index: number;
+    static deserialize(pData: number): WindowManagerAddViewParams;
```

Comment:
`deserialize` sounds a bit generic to me, I would have put emphasis on the fact that its for marshaling

Reply:
Had the same thought :)